src/geom/ribbon.jl

```diff
@@ -10,41 +10,42 @@ RibbonGeometry(default_statistic=Gadfly.Stat.identity(); fill=true, tag=empty_ta
     Geom.ribbon
 
 Draw a ribbon at the positions in `x` bounded above and below by `ymax` and
-`ymin`, respectively.  Optionally draw multiple ribbons by grouping with `color`.
+`ymin`, respectively.  Optionally draw multiple ribbons by grouping with `color` and `alpha` or `linestyle`.
```

Comment:
what does "color and alpha or linestyle" mean?  the boolean logic is unclear

Reply:
I've clarified this. It differs whether the ribbon is filled or not filled:
https://github.com/GiovineItalia/Gadfly.jl/blob/99f54e467e92c6478dea5ef3ec68ae4bfe30377e/src/geom/ribbon.jl#L75-L76